lib/rspec/rails/matchers/active_job.rb

```diff
@@ -67,7 +67,7 @@ def thrice
           end
 
           def failure_message
-            "expected to enqueue #{base_message}".tap do |msg|
+            "expected to #{self.class::FAILURE_MESSAGE_EXPECTATION_ACTION} #{base_message}".tap do |msg|
```

Comment:
Is this scoping definitely necessary?

Reply:
Yes, it ends up with an error otherwise:
```
NameError: uninitialized constant RSpec::Rails::Matchers::ActiveJob::Base::FAILURE_MESSAGE_EXPECTATION
_ACTION
```

Reply:
:👍: